lib/js/index.ts

```diff
@@ -7,6 +7,7 @@ export { default as Button } from './components/Buttons/Button';
 export * from './components/Buttons/Button/Button.consts';
 export { default as Card } from './components/Cards/Card/';
 export { default as CardExpandable } from './components/Cards/CardExpandable/';
+export * from './components/Divider';
```

Comment:
does it export `default`?

Reply:
https://github.com/bethinkpl/design-system/blob/a6c4c40bd80acb563c97186584974c4eee298d44/lib/js/components/Divider/index.ts#L1-L4